### PR TITLE
Add Rapu Ture

### PR DESCRIPTION
**Pull request by maukoquiroga**
Capture d’écran 2019-11-23 à 15 41 16

### CODE REVIEW

File: content/_index.en.md [hunk 1]
```diff
@@ -1,26 +1,27 @@
...
     description: Compare the law now to what it could be.
     caption_1: of the French Movement for a Basic Income that uses OpenFisca and its
     caption_2: to evaluate the cost and benefits of a reform introducing a basic income in France.
-get_started: Get started
-products: Powered by OpenFisca
+
+  rapu_ture:
+    title: Explore government rules
+    description: Model any kind of government rule, explore them and evaluate who is eligible to what.
+    caption: uses OpenFisca to allow New Zealand citizens, civil servants, policy officers and developers to explore government rules.
```
Comment (MattiSG) [inline content/_index.en.md]: I'd suggest either
```suggestion
    caption: uses OpenFisca to allow New Zealanders to explore government rules.
```
or
```suggestion
    caption: uses OpenFisca to allow Aotearoa New Zealand citizens, civil servants, policy officers and developers to explore government rules.
```

**Comment by MattiSG** [comment]
Excited about all these changes, thanks @maukoquiroga for having acted so promptly on our discussions! 😃

File: content/_index.en.md [hunk 2]
```diff
@@ -1,26 +1,27 @@
 ---
 title: "Turn law into software"
+get_started: Get started
+products: Powered by OpenFisca
+
 usecases:
   mes_aides:
     title: Calculate citizens' rights
-    description: From an individual situation, calculate any rule of a national tax and benefit system.
+    description: From an individual situation, calculate any rule of a tax and benefit system.
```
Comment (Br3nda) [inline content/_index.en.md]:
```suggestion
    description: Web interface to explore variables, formulas, paramenters and scenarios (unit tests) in New Zealand's Openfisca built from government legislation, regulation and some policies.
```

File: content/_index.en.md [hunk 3]
```diff
@@ -1,26 +1,27 @@
...
     title: Evaluate reforms impact
     description: Compare the law now to what it could be.
     caption_1: of the French Movement for a Basic Income that uses OpenFisca and its
     caption_2: to evaluate the cost and benefits of a reform introducing a basic income in France.
-get_started: Get started
-products: Powered by OpenFisca
+
+  rapu_ture:
+    title: Explore government rules
+    description: Model any kind of government rule, explore them and evaluate who is eligible to what.
```
Comment (Br3nda) [inline content/_index.en.md]:
```suggestion
    description: Web interface to explore variables, formulas, parameters and scenarios (unit tests) in New Zealand's Openfisca built from government legislation, regulation and some policies.
```

File: content/_index.en.md [hunk 4]
```diff
@@ -1,26 +1,27 @@
...
   reforms:
     title: Evaluate reforms impact
     description: Compare the law now to what it could be.
     caption_1: of the French Movement for a Basic Income that uses OpenFisca and its
     caption_2: to evaluate the cost and benefits of a reform introducing a basic income in France.
-get_started: Get started
-products: Powered by OpenFisca
+
+  rapu_ture:
+    title: Explore government rules
```
Comment (Br3nda) [inline content/_index.en.md]:
```suggestion
    title: Rapu Ture (Exploring  the Rules)
```

File: content/_index.en.md (hunk 1 above)
Comment (maukoquiroga) [reply content/_index.en.md]: Used the second one.

**Comment by maukoquiroga** [comment]
Hi @Br3nda ! Thanks for your suggestions. After integrating them, I realised that, currently, seems we're using the `caption` for a general OpenFisca feature, and `description` for the description of the showcased application.

It is definitely not ideal nor semantic, however I propose we stick to that structure for this pull request at least, to avoid having to change all others as well (which we could do in another pull request of course).

So I moved your proposed text from `caption` to `description`, What do you think?

Capture d’écran 2019-12-27 à 02 29 44

**Comment by Br3nda** [comment]
Adding @merxplat, who is the new tech lead for the NZ team.

**Pull request merged**